src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/"
+      data-qa="big"
+      method="get"
+      name="bar"
+      class="search__form"
+    >
+      <label class="container">
+        <img
+          src="images/Search.svg"
+          alt="image"
+          class="logo"
+        >
+        <input
+          type="search"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+          class="search search_bar"
```

Comment:
You dont need there 2 classes<img width="1041" alt="Screenshot 2022-09-20 at 16 26 12" src="https://user-images.githubusercontent.com/104769168/191269942-22e9e8c8-79bf-4051-ba18-0b09ab7038c0.png">
```suggestion
          class="search"
```